include/seastar/core/reactor.hh

```diff
@@ -205,6 +204,7 @@ public:
     private:
         friend class file_data_source_impl;
         friend void io_completion::complete_with(ssize_t);
+        friend class io_queue;
         static io_stats& local() noexcept;
```

Comment:
This should be inlined to restore fast access.

Reply:
It's inlined when defined

https://github.com/scylladb/seastar/blob/66ecd34f2aff7607814a26432a02a441e8d25403/include/seastar/core/reactor.hh#L777-L779